package/samplers/grey_wolf_optimization/README.md

```diff
@@ -0,0 +1,48 @@
+---
+author: k-onoue
+title: Sampler based on Grey Wolf Optimization (GWO)
```

Comment:
I don't have a strong opinion, but I think including 'Grey Wolf Optimizer' as a prefix in the title could have a positive impact on visibility in the feature list, potentially improving the ability to find relevant information.

But we can work on it in a follow-up PR.

```suggestion
title:  Grey Wolf Optimization (GWO) Sampler
```

<img width="1334" alt="image" src="https://github.com/user-attachments/assets/da46865a-79de-4b8c-98fd-7f06431fa38a">